src/const.js

```diff
@@ -122,6 +122,13 @@ export const MIME_TO_FILE_EXTENSIONS = {
 // List of the mime types for the allowed static theme images.
 export const STATIC_THEME_IMAGE_MIMES = Object.keys(MIME_TO_FILE_EXTENSIONS);
 
+// List of deprecated static theme's LWT aliases.
+export const STATIC_THEME_LWT_ALIASES = [
```

Comment:
@nt1m do you mind to confirm if these are all the static theme manifest properties that has to be marked as deprecated LWT aliases?

Thanks in advance for your help!